include/stout/flags/flags.cc

```diff
@@ -587,18 +560,36 @@ void Parser::Parse(const std::vector<ArgumentInfo>& values) {
   }
 
   // Ensure required flags are present.
-  for (const auto& [_, field] : fields_) {
+  for (const auto& [name, field] : fields_) {
     const stout::v1::Flag& flag =
         field->options().GetExtension(stout::v1::flag);
 
-    // This field is not required, so just continue.
-    if (!flag.required())
+    // The field has been parsed, all good, just continue.
+    if (parsed_.count(field) > 0) {
       continue;
+    }
 
-    // Required field has been parsed, so it is present.
-    // All good, just continue.
-    if (parsed_.count(field) > 0)
+    // Need to normalize string default value before parsing.
+    const std ::string normalized_value =
+        (field->type() != google::protobuf::FieldDescriptor::TYPE_STRING)
+        ? flag.default_()
```

Comment:
Why does `default_` have a trailing underscore here? I'd expect just `default` since that's the field name. (Is `default` some sort of reserved word?)

Please add a comment.